lib/write.js

```diff
@@ -112,7 +112,9 @@ exports.usingBmap = (deviceFileDescriptor, options = {}) => {
  */
 exports.usingStreaming = (deviceFileDescriptor, options = {}) => {
   return new Bluebird((resolve, reject) => {
-    const checksumStream = new CRC32Stream();
+    const checksumStream = new ChecksumStream({
+      algorithms: options.checksumAlgorithms
```

Comment:
If (after all the other checks) `options.checksumAlgorithms` ends up empty, would it be worth skipping the `ChecksumStream` entirely? Or is that an over-optimisation?